os/arch/arm/src/amebad/amebad_serial.c

```diff
@@ -114,7 +114,7 @@
 #elif defined(CONFIG_UART1_SERIAL_CONSOLE)
 #define CONSOLE_DEV             g_uart1port             /* UART1 is console */
 #define TTYS0_DEV               g_uart1port             /* UART1 is ttyS0 */
-#define CONSOLE                 UART1_DEV
+#define CONSOLE                 UART3_DEV
```

Comment:
All other defines are related to uart1, but this define is uart3.
Is it correct? And if then, why we should set like this?

Reply:
What we tested does not include this change. Could you let me know why this is suddenly added?

Reply:
@sunghan-chang In our test code, I missed this change....
@AdamZhang0124 , Anyway, in our test, we can successfully run without this change.
Is this needed?

Reply:
Hi @sunghan-chang  @jeongchanKim,
This is an notation mismatch I recently found.
The TizenRT uart1 actually maps to Ameba's uart3.
It won't affect the function.

Reply:
Thanks for checking :)